### PR TITLE
prevent xss attacks through field values

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,6 +121,7 @@
     "nsp": "nsp check",
     "pretest": "npm run hint",
     "test": "npm run test:unit && npm run coverage",
+    "test:unitlocal": "mocha --compilers js:babel-core/register --require babel-polyfill --reporter-options mochaFile=$(bash scripts/test_output_dir.sh junit)/junit.xml test",
```
Comment (xdamman) [inline package.json]: can we use `test:unit:dev` to match `npm run db:migrate:dev` to apply to local dev environment?

File: server/middleware/sanitizer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    if (req.body) {
+      req.body = sanitizeHelper(req.body);
+    }
+    return next();
+  };
+};
+
+const sanitizeHelper = (value) => {
+  if (typeof value === 'string') {
+    //value = value.replace(/<script>.*<\/script>/); // remove all script tags and anything in the middle.
```
Comment (xdamman) [inline server/middleware/sanitizer.js]: why is this commented out?

File: server/middleware/sanitizer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    //value = value.replace(/<.*?>/gi, ''); // for other tags, only remove the tags
+    value = value.replace(/&gt;/gi, '>');
+    value = value.replace(/&lt;/gi, '<');
```
Comment (xdamman) [inline server/middleware/sanitizer.js]: why are we replacing html encoded with their equivalent?
isn't it still a problem?

Comment (xdamman) [inline server/middleware/sanitizer.js]: tests pass so I guess it's fine, I just want to understand.

Comment (asood123) [inline server/middleware/sanitizer.js]: value.replace replaces `&gt;` tags with `>` and then html sanitizer in the next step removes all html tags

File: server/middleware/sanitizer.js (hunk 2 above)
Comment (asood123) [inline server/middleware/sanitizer.js]: argh will fix

**Pull request merged**